runtime/moonbeam/tests/integration_test.rs

```diff
@@ -2770,6 +2772,44 @@ fn test_xcm_utils_ml_to_account() {
 	});
 }
 
+#[test]
+fn test_xcm_utils_weight_message() {
+	ExtBuilder::default().build().execute_with(|| {
+		let xcm_utils_precompile_address = H160::from_low_u64_be(2060);
+		let expected_weight: Weight = UnitWeightCost::get();
+
+		let message: Vec<u8> = xcm::VersionedXcm::<()>::V2(Xcm(vec![ClearOrigin])).encode();
+
+		let input = XcmUtilsPCall::weight_message {
+			message: message.into(),
+		};
+
+		Precompiles::new()
+			.prepare_test(ALICE, xcm_utils_precompile_address, input)
+			.expect_cost(0)
+			.expect_no_logs()
+			.execute_returns(EvmDataWriter::new().write(expected_weight).build());
```

Comment:
Prefer to use `execute_returns_encoded` ?

Reply:
fixed in fc4e9d8258c9f8c388cb12b2e06fb6cbcb97157f